scripts/update-i18n.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -e
+
+PWD=$(dirname "${BASH_SOURCE[0]}")
+cd $PWD/..
+
+if [ ! -d .tx ]; then
+    echo "please make sure this script is under the scripts directory!"
+    exit -9
+fi
+
+if [ "$1" != "-v" ]; then
+    QUIET_FLAGS="-silent"
+fi
+
+function regenerate_source() {
+    SEAFILE_PROJECT="seafile-client.pro"
+    rm -f $SEAFILE_PROJECT
+    qmake -project -o $SEAFILE_PROJECT
+    lrelease $SEAFILE_PROJECT $QUIET_FLAGS
+    lupdate $SEAFILE_PROJECT $QUIET_FLAGS
+    rm -f $SEAFILE_PROJECT
+}
```

Comment:
I think `lupdate` requires the qmake project file in some degree. and I believe whether the qmake project file exists or not makes difference. And in fact, many of sources of translations hadn't updated since months ago.

Sadly, I didn't look into the issue any further since the solution (regenerate the pro file and execute lrelease and lupdate) solves this issue. So I wrote this script not only for the automatizing the translations-updating progress, but for recording this solution we found (sometime we might forget how to solve the issues we faced before, we haven't to need to solve them again because we can everything from this script)